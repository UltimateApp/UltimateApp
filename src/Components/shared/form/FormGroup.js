import React from 'react';
import { StyleSheet, View } from 'react-native';

const FormGroup = (props) => {
  return <View style={styles.formGroup}>{props.children}</View>;
};

export default FormGroup;

const styles = StyleSheet.create({
  formGroup: {
    marginBottom: 10,
  },
});
