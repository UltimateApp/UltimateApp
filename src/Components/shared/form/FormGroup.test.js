import React from 'react';
import renderer from 'react-test-renderer';
import { View } from 'react-native';

import FormGroup from './FormGroup';

describe('FormGroup', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <FormGroup>
          <View />
        </FormGroup>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
