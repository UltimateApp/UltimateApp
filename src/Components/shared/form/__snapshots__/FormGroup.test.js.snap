// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormGroup renders correctly 1`] = `
<View
  style={
    Object {
      "marginBottom": 10,
    }
  }
>
  <View />
</View>
`;
